docs/source/app-dev/services.rst

```diff
@@ -94,6 +94,17 @@ The command submission service deduplicates submitted commands based on their :r
 
 For details on how to use command deduplication, see the :ref:`Application Architecture Guide <command-deduplication>`.
 
+.. note:: Ledgers with command deduplication across participants
+
+  - The ledger server may extend the deduplication period specified in the request arbitrarily, up until the maximum deduplication duration specified in the :ref:`ledger configuration <ledger-configuration-service>`.
```

Comment:
```suggestion
  - The ledger may extend the deduplication period specified in the request arbitrarily, up until the maximum deduplication duration specified in the :ref:`ledger configuration <ledger-configuration-service>`.
```

Reply:
Do we want to capture that it also accounts for time skews? Currently we basically account for `min_record_time` by adding the `min_skew` to calculate the deduplicate until